manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
```

Comment:
validate_bool will cause the apply to fail if a boolean isn't passed, but 'if str2bool' is being used as a shortcut for the string passed to the if statement. I could explicitly write out 'if $cron_purge == \true|True|yes|Yes\ {' but str2bool takes care of that.